operations/boot_orchestration/Options.md

```diff
@@ -53,13 +53,13 @@ The following are the BOS global options:
 
     The amount of time in seconds BOS will wait for a response from BSS to a request. After this time, the request will
     time out. The default is 10 seconds.
-    Note: This option is only available as a 'hotfix' in CSM 1.5.
+    Note: This option is added in CSM 1.5.3; in earlier versions of CSM 1.5, it is only available in a hotfix.
```

Comment:
For all of these, I am wondering if "it is only available _as_ a hotfix" (rather than "in" makes more sense?

Reply:
Works for me. Change made.